test/test_client_errors.rb

```diff
@@ -169,6 +169,26 @@ def test_normalize_error_path
       ]
     }
     assert_equal expected, actual
+
+    actual = {
+      "data" => nil,
+      "errors" => [
+        {
+          "message" => "error",
+          "path" => nil
+        }
+      ]
+    }
+    GraphQL::Client::Errors.normalize_error_paths(actual["data"], actual["errors"])
+    expected = {
+      "data" => nil,
+      "errors" => [
+        {
+          "message" => "error",
+          "normalizedPath" => %w(data)
+        }
+      ]
+    }
```

Comment:
You created two variables, but never asserted anything about them.

```suggestion
    }
    assert_equal expected, actual
```

Reply:
You are right :sweat_smile: ! Thank you. I will fix this.